src/directives/click-outside.js

```diff
@@ -1,31 +1,44 @@
 function directive (e, el, binding, v) {
+  if (!v.context.isActive) return
   let cb = () => true
 
   if (binding.value) cb = binding.value
 
-  if (v.context.isActive &&
-    (e && e.target) &&
-    (e.target !== el && !el.contains(e.target)) &&
-    cb(e)
-  ) {
-    v.context.isActive = false
+  const elements = (binding.include || (() => []))()
+  elements.push(el)
+
+  if (!clickedInEls(e, elements) && cb(e)) {
+    setTimeout(() => v.context.isActive = false, 0)
```

Comment:
Not sure why lint didn't pick this up, but functions returning assignment should be wrapped in parenthesis